### PR TITLE
Added Card Like structure in experience section

### PR DESCRIPTION
**Pull request by Vyomrana02**
Output for same is
image

### CODE REVIEW

**Comment by saiteja13427** [comment]
@Vyomrana02 can you add job description as well to the cards (as commented on the issue)

Also can you fix the build warnings so that we can see a deploy preview.

**Comment by ashutosh1919** [comment]
@Vyomrana02 why is the workflow failing? @saiteja13427, can you please suggest some changes so that it is successful?

**Comment by saiteja13427** [comment]
@ashutosh1919 it is failing due to some build warnings. As we have CI=true, it considers warnings also as errors and fails.

Thus, asked to fix the build warnings.

File: src/components/experienceCard/ExperienceCard.js [hunk 1]
```diff
@@ -1,68 +1,137 @@
-import React, { Component } from "react";
-import "./ExperienceCard.css";
+import React, { Component } from 'react';
+import './ExperienceCard.css';
+import { Fade } from "react-reveal";
+import ProjectLanguages from '../projectLanguages/ProjectLanguages';
```
Comment (ashutosh1919) [inline src/components/experienceCard/ExperienceCard.js:4]:
```suggestion
```

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
Removed unused code

**Comment by ashutosh1919** [comment]
@Vyomrana02 Please add a description to the experience cards. Also, please increase the width of the card to full width of accordian.

**Comment by Vyomrana02** [comment]
This will work??

image

**Comment by saiteja13427** [comment]
@Vyomrana02 Looks good, maybe decrease the gap above the description a little bit and push the code so that we can check the deploy preview!

**Comment by Vyomrana02** [comment]
Hello I am facing issue with the vertical line size and content size in card.
card grows or shrinks acc. to content but the height of vertical line is fixed so sometimes overlap or makes space.
1 Option to solve this:-
1. setting the constant height of cards for all so no overlapping or space .
 what should I do??

**Comment by ashutosh1919** [comment]
@Vyomrana02 please push the code so that we can have a look. @saiteja13427 , please help him once he pushes the code.

**Comment by saiteja13427** [comment]
Yes @Vyomrana02 Please push the code, Let me look into it and find a solution!

**Comment by Vyomrana02** [comment]
dynamic size of card output
image

I have pushed into this fork :-
https://github.com/Vyomrana02/masterPortfolio
changes in experiencecard.js

**Comment by saiteja13427** [comment]
@Vyomrana02 Why did you close this PR?
Can you open a new fresh PR with only the experience card changes, so that we can check the deployment preview?

**Pull request closed without merging**